rest-kraken/src/main/java/com/github/twitch4j/kraken/TwitchKraken.java

```diff
@@ -33,6 +33,56 @@
         }
     }).get();
 
+    /**
+     * The default baseUrl to pass to {@link #getGlobalBadges(URI, String)} and {@link #getChannelBadges(URI, String, String)}
+     */
+    @Unofficial
+    URI BADGES_BASE_URL = ((Supplier<URI>) () -> {
+        try {
+            return new URI("https://badges.twitch.tv/v1");
+        } catch (Exception e) {
+            return null;
+        }
+    }).get();
+
+    @Unofficial
+    @RequestLine("GET /badges/global/display?language={language}")
+    HystrixCommand<BadgeSets> getGlobalBadges(
```

Comment:
Considering we're on java 8 i don't think we can make the helper methods private/not expose them - maybe we can deprecate them to discourage users from trying to use them?